### PR TITLE
Wcag notes

### PR DESCRIPTION
**Pull request by mhewson**

### CODE REVIEW

File: docs/components/links/README.md [hunk 1]
```diff
@@ -198,10 +201,12 @@ To ensure that usage of this component complies with accessibility guidelines:
...
-  - Ensure that each link can be accessed using the keyboard. Don’t manipulate the default tab index
+  - Always provide a `href` attribute. Empty `href` attributes are not considered true links and are not accessible to keyboard users.
   - When using images as links, the `alt` attribute acts as the link text. Describe the action or hyperlink that the image represents
-
+- Ensure that if the keyboard focus can be moved to a component of the page using a keyboard interface, then focus can be moved away from that component using only a keyboard interface, and, if it requires more than unmodified arrow or tab keys or other standard exit methods, the user is advised of the method for moving focus away.
+-  Use `aria-label` or `aria-labelledby` to provide a more descriptive name when the text content of the link cannot be changed.
+- The purpose of each link MUST be understandable and distinguishable from other links on the same page, either from the link text alone (ideally), or from the immediate surrounding context of the link.
+- Links to the same destinations MUST be consistently identified with the same (or very similar) link text across all pages of the site.
```
Comment (cowills) [inline docs/components/links/README.md:209]: this one appears on line 192 as well, duplicate?

File: docs/components/rating/README.md [hunk 2]
```diff
@@ -188,16 +188,19 @@ Change size for the star icon and text. Default size is medium.
 
 ## Accessibility
 
-To ensure that usage of this component complies with accessibility guidelines:
+Many WCAG requierments are contextual to their implementation. 
+To ensure that usage of this component complies with accessibility guidelines you are responcible for the following:
 
-- Use Rating component only on light backgrounds
+- Always provide a `href` attribute. Empty `href` attributes are not considered true links and are not accessible to keyboard users.
```
Comment (cowills) [inline docs/components/rating/README.md:194]: ? href is not required in our rating component, is this just copy pasta from the links section or should we fix that in rating?

File: docs/components/rating/README.md [hunk 3]
```diff
@@ -188,16 +188,19 @@ Change size for the star icon and text. Default size is medium.
...
 
 <br/>
 
 This component has compliance with WCAG guidelines by:
 
-- Providing screen reader text that reads “Rated { rounded } out of 5 with { count }”
+- Providing screen reader text customized to each available variant”
 - Using text color with a Level AA contrast ratio of 4.5:1 contrast between the text color and the background (only when displayed on light backgrounds)
+- Replacing icon stars with utf-8 characters when images are disabled 
+-
```
Comment (cowills) [inline docs/components/rating/README.md:203]: blank entry here

File: docs/components/rating/README.md (hunk 2 above)
Comment (mhewson) [reply docs/components/rating/README.md:194]: this requirement can be articulated better - "when linking the rating component be sure to provide a valid href value" or something

**Pull request merged**